tests/core/pyspec/eth2spec/test/capella/block_processing/test_process_withdrawals.py

```diff
@@ -872,6 +879,7 @@ def test_partially_withdrawable_validator_legacy_max_minus_one(spec, state):
     set_eth1_withdrawal_credential_with_balance(
         spec, state,
         validator_index,
+        effective_balance=spec.MAX_EFFECTIVE_BALANCE - spec.EFFECTIVE_BALANCE_INCREMENT,
         balance=spec.MAX_EFFECTIVE_BALANCE - 1
```

Comment:
This should still be `spec.MAX_EFFECTIVE_BALANCE` so that we're not testing multiple things at once. We want the second check in the function to fail. See:

https://github.com/ethereum/consensus-specs/blob/c95d1df58fb185c5c96773be88dd6fc15225bf1f/specs/capella/beacon-chain.md?plain=1#L284-L290